MAINTAINERS

```diff
@@ -2,6 +2,14 @@ The maintainers are generally available in Slack at
 https://cloud-native.slack.com in #flux (https://cloud-native.slack.com/messages/CLAJ40HV3)
 (obtain an invitation at https://slack.cncf.io/).
 
+These maintainers are shared with other Flux v2-related git
+repositories under https://github.com/fluxcd, as noted in their
+respective MAINTAINERS files.
+
+For convenience, they are reflected in the GitHub team
+@fluxcd/flux-v2-maintainers -- if the list here changes, that team
```

Comment:
```suggestion
@fluxcd/flux2-maintainers -- if the list here changes, that team
```

to match the repository name?

Reply:
Ah, dilemma -- I named it for the project ("Flux v2"), but it does correspond to the maintainers in this repo.

Reply:
I'll change it to flux2-maintainers, saves a needless conversation.